cpp/include/cudf/rolling.hpp

```diff
@@ -58,7 +58,7 @@ std::unique_ptr<column> rolling_window(
   size_type preceding_window,
   size_type following_window,
   size_type min_periods,
-  std::unique_ptr<aggregation> const& agg,
+  std::unique_ptr<rolling_aggregation> const& agg,
```

Comment:
While we're at it, can we change this too? People are already going to have to update their calls to this API as it is. Might as well make it so they only have to change it once. 
```suggestion
  rolling_aggregation const& agg,
```